spec/features/ncr/work_orders/approve_spec.rb

```diff
@@ -21,6 +21,21 @@
     end
   end
 
+  scenario "allows delegate to approve work order" do
+    Timecop.freeze do
```

Comment:
Fine to keep in other specs, I guess my question is: does this test pass without freezing time? And if so, should we write it without time freezing?